asketch2sketch.sketchplugin/Contents/Sketch/asketch2sketch.js

```diff
@@ -0,0 +1,2273 @@
+var that = this;
```

Comment:
not sure how this file ended up here 🤔 but plugin files are now in the `/build/` folder and are not included in the repo (gitignored)